lib/ipam/ipam_test.go

```diff
@@ -600,6 +600,100 @@ var _ = testutils.E2eDatastoreDescribe("IPAM tests", testutils.DatastoreAll, fun
 		})
 	})
 
+	Describe("IPAM IP borrowing", func() {
+		node1 := "host1"
+		node2 := "host2"
+
+		It("IP borrowing and StrictAffinity", func() {
```

Comment:
nit: generally, It() statements should read like a sentence describing what the test is checking.

e.g., `It("should respect IPAMConfig.StrictAffinity when it is changed")`